test/elixir_analyzer/exercise_test/two_fer_test.exs

```diff
@@ -0,0 +1,260 @@
+      status: :disapprove,
+      comments_include: [Constants.two_fer_use_guards()] do
+      defmodule TwoFer do
+        def two_fer(name \\ "you") do
+          "One for #{name}, one for me"
+        end
+      end
+    end
+
+    test_exercise_analysis "refer when is_binary or is_bitstring in used outside of the function head",
+      status: :refer,
+      comments_include: [Constants.two_fer_use_function_level_guard()] do
+      [
+        defmodule TwoFer do
+          def two_fer(name \\ "you") do
+            is_binary(name)
+            "One for #{name}, one for me"
+          end
+        end,
+        defmodule TwoFer do
+          def two_fer(name \\ "you") do
+            if is_bitstring(name) do
+              "One for #{name}, one for me"
+            else
+              raise FunctionClauseError
+            end
+          end
+        end
+      ]
+    end
+  end
+
+  describe "using auxiliary functions" do
+    test_exercise_analysis "multiple implementations of two_fer/1 don't count as auxiliary functions",
+      comments_exclude: [Constants.two_fer_use_of_aux_functions()] do
+      defmodule TwoFer do
+        def two_fer("foo") do
+          "One for foo, one for me"
+        end
+
+        def two_fer(name) do
+          "One for #{name}, one for me"
+        end
+      end
+    end
+
+    test_exercise_analysis "refer there are other functions than two_fer/1",
+      comments_include: [Constants.two_fer_use_of_aux_functions()],
+      status: :refer do
+      [
+        defmodule TwoFer do
+          def two_fer(name \\ "you") do
+            "One for #{name}, one for me"
+          end
+
+          defp foo() do
+          end
+        end,
+        defmodule TwoFer do
+          def two_fer(name \\ "you") do
+            "One for #{name}, one for me"
+          end
+
+          defp foo(), do: nil
+        end,
+        defmodule TwoFer do
+          def two_fer(name \\ "you") do
+            "One for #{name}, one for me"
+          end
+
+          defp foo(_a), do: nil
+        end,
+        defmodule TwoFer do
+          def two_fer(name \\ "you") do
+            "One for #{name}, one for me"
+          end
+
+          defp foo(_a, _b), do: nil
+        end
+        #        defmodule TwoFer do
+        #          def two_fer(name \\ "you") do
+        #            "One for #{name}, one for me"
+        #          end
+        #
+        #          def foo(_a, _b), do: nil
+        #        end,
+        #        defmodule TwoFer do
+        #          def foo(_a, _b, _c, _d), do: nil
+        #
+        #          def two_fer(name \\ "you") do
+        #            "One for #{name}, one for me"
+        #          end
+        #        end,
+        #        defmodule TwoFer do
+        #          def foo(_a) do
+        #          end
+        #
+        #          def two_fer(name \\ "you") do
+        #            "One for #{name}, one for me"
+        #          end
+        #        end
```

Comment:
I would like for those test cases to also pass, but currently they fail due to: https://github.com/exercism/elixir-analyzer/pull/24/files#r512786752